archetypes.js

```diff
@@ -0,0 +1,1484 @@
+const emptyBuild = {
+  position: 'None',
+  shortPosition: 'None',
+  archetype: 'None',
+  baseTpe: 0,
+  strengthMin: 0,
+  strengthMax: 0,
+  agilityMin: 0,
+  agilityMax: 0,
+  armMin: 0,
+  armMax: 0,
+  intelligenceMin: 0,
+  intelligenceMax: 0,
+  throwingAccuracyMin: 0,
+  throwingAccuracyMax: 0,
+  tacklingMin: 0,
+  tacklingMax: 0,
+  speedMin: 0,
+  speedMax: 0,
+  handsMin: 0,
+  handsMax: 0,
+  passBlockingMin: 0,
+  passBlockingMax: 0,
+  runBlockingMin: 0,
+  runBlockingMax: 0,
+  enduranceMin: 0,
+  enduranceMax: 0,
+  kickPowerMin: 0,
+  kickPowerMax: 0,
+  kickAccuracyMin: 0,
+  kickaccuracyMax: 0
+};
+
+const safetyCenterFielder = {
+  position: 'Safety',
+  shortPosition: 'S',
+  archetype: 'Center Fielder',
+  baseTpe: 466,
```

Comment:
shouldn't it be 451? (401 from the template + 50 available)?

Reply:
(MAX: 70) Strength: 40 (X TPE)
(MAX: 95) Agility: 70 (X TPE)
(MAX: 25) Arm: 1 (X TPE)
(MAX: 80) Intelligence: 40 (X TPE)
(MAX: 25) Throwing Accuracy: 1 (X TPE)
(MAX: 75) Tackling: 50 (X TPE)
(MAX: 100) Speed: 70 (X TPE)
(MAX: 75) Hands: 55 (X TPE)
(MAX: 25) Pass Blocking: 1 (X TPE)
(MAX: 25) Run Blocking: 1 (X TPE)
(MAX: 100) Endurance: 70 (X TPE)
(MAX: 25) Kick Power: 1 (X TPE)
(MAX: 25) Kick Accuracy: 1  (X TPE)